packages/publisher/electron-release-server/package.json

```
@@ -9,7 +9,9 @@
   "typings": "dist/PublisherERS.d.ts",
   "devDependencies": {
     "chai": "4.2.0",
-    "mocha": "^7.1.0"
+    "mocha": "^7.1.0",
+    "proxyquire": "^2.1.3",
+    "fetch-mock": "^8.1.0"
```

FYI, the tests are currently failing because I merged this PR recently: https://github.com/electron-userland/electron-forge/pull/1534
```suggestion
    "fetch-mock": "^9.0.0"
```